### PR TITLE
Dropcols

### PR DESCRIPTION
**Pull request by timcadman**

### CODE REVIEW

File: R/make-age-polys.R [hunk 1]
```diff
@@ -53,7 +53,7 @@ dh.makeAgePolys <- function(df = NULL, age_var = NULL,
   }
 
   if (checks == TRUE) {
-    .isDefined(df = df, vars = vars, conns = conns)
+    .isDefined(df = df, vars = age_vars, conns = conns)
```
Comment (ESCRI11) [inline R/make-age-polys.R:56]:
```suggestion
    .isDefined(df = df, vars = age_var, conns = conns)
```

`age_vars` seems not to be defined

File: R/drop-cols.R [hunk 2]
```diff
@@ -57,33 +57,73 @@ dh.dropCols <- function(df = NULL, vars = NULL, new_obj = df, type = NULL,
...
+        times.transmit = paste0(.x),
+        length.out.transmit = "NA", 
+        each.transmit = "1",
+        x1.includes.characters = FALSE,
+        source.x1 = "clientside", 
+        source.times = "clientside",
+        source.length.out = "clientside",
+        source.each = "clientside")
+      
+      DSI::datashield.assign(conns[.y], "ones", calltext)
```
Comment (ESCRI11) [inline R/drop-cols.R:126]:
```suggestion
      DSI::datashield.assign(conns[.y], "ONES", calltext)
```

On the lines https://github.com/lifecycle-project/ds-helper/pull/119/files#diff-068b87ce02bc1aa5a59f899bec8f682e881db91f0abff100bf03c599bf92113bR75-R76 and https://github.com/lifecycle-project/ds-helper/pull/119/files#diff-068b87ce02bc1aa5a59f899bec8f682e881db91f0abff100bf03c599bf92113bR88-R89, the object `"ONES"` will not be found.

**Review by ESCRI11 — COMMENTED** [review]
Added a couple suggestions. Please review them.

File: R/make-age-polys.R (hunk 1 above)
Comment (timcadman) [reply R/make-age-polys.R]: Thanks - DONE

File: R/drop-cols.R (hunk 2 above)
Comment (timcadman) [reply R/drop-cols.R]: Thanks - DONE

**Review by sidohaakma — APPROVED** [review]

**Pull request merged**